docs/client/README.md

```diff
@@ -0,0 +1,15 @@
+# Semantic Conventions For Client SDKs
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines general semantic conventions for client-side applications.
+The most common types of client applications are web applications (that run in a web browser)
+and mobile apps (that run on a phone or tablet).
+Client applications are differentiated from server applications in that they service a user rather than requests.
```

Comment:
Based on  [this comment](https://github.com/open-telemetry/semantic-conventions/pull/215#discussion_r1276731735) by @bryce-b, does it make sense to provide a definition of "user"? (_person interacting with the interface_: does this exclude synthetic users?)